pkg/orchestrator/gitHubActions.go

```diff
@@ -99,7 +99,11 @@ func (g *GitHubActionsConfigProvider) GetLog() ([]byte, error) {
 		i := i // https://golang.org/doc/faq#closures_and_goroutines
 		wg.Go(func() error {
 			_, resp, err := g.client.Actions.GetWorkflowJobLogs(g.ctx, g.owner, g.repo, jobs[i].ID, true)
-			if err != nil {
+			if err != nil && err.Error() != "unexpected status code: 200 OK" {
```

Comment:
does github say `unexpected status code: 200 OK` . that is quite confusing to 200 and then say unexpected status ?

Reply:
it's not GitHub itself but API client implementation for golang

Reply:
I've checked newer versions of this sdk. Same thing there too

Reply:
In short, when trying to fetch logs GitHub API first returns 301 redirect response together with URL to download the actual logs. When you send GET request to this URL, you receive 200 OK response with plain text logs, but GetWorkflowJobLogs implementation from sdk is strange, that's why we get **200 OK** status as error. It follows redirect response, downloads logs and expects another redirect response instead of 200 OK status.

Reply:
but do we need and `&&` condition, could'nt there be any other generic error would'nt `||`  be safer ?

Reply:
Yes, you're right. Missed that, my bad. Fixed